cmd/mongodb-healthcheck/main.go

```diff
@@ -105,13 +106,13 @@ func main() {
 		}
 		log.Debug("Member passed DC/OS readiness check")
 	case "k8s liveness":
-		log.Debug("Running Kubernetes liveness check")
-		state, memberState, err := healthcheck.HealthCheck(session, healthcheck.OkMemberStates)
+		log.Info("Running Kubernetes liveness check")
+		state, memberState, err := healthcheck.HealthCheckLiveness(session, healthcheck.OkMemberStates, int64(*startupDelaySeconds))
```

Comment:
why we need to return `state` if we do "exit 1" only if `err != nil`?

Reply:
just to log it here https://github.com/percona/mongodb-orchestration-tools/pull/247/files#diff-f42babb5e01b13430e679631c2d5a35aR116

Reply:
we log `memberState`, why we need `state`?

Reply:
oh, it is used in `os.Exit(state.ExitCode())`

Reply:
Done https://github.com/percona/mongodb-orchestration-tools/pull/247/commits/0625196a2eacfcb90feed8c91400190bba1a4e02